frontend/tests/features/wheresMyGene.test.ts

```diff
@@ -60,25 +66,13 @@ describe("Where's My Gene", () => {
     await expect(page).toHaveSelector(getText("STEP 3"));
     await expect(page).toHaveSelector(getText("Explore Gene Expression"));
 
-    // Filters Panel
-    // (thuang): `*` is for intermediate match
-    // https://playwright.dev/docs/selectors#intermediate-matches
-
-    async function getTissueSelectorButton() {
-      return page.$(getTestID(ADD_TISSUE_ID));
-    }
-
-    async function getGeneSelectorButton() {
-      return page.$(getTestID(ADD_GENE_ID));
-    }
-
-    await clickUntilOptionsShowUp(getTissueSelectorButton, page);
+    await clickUntilOptionsShowUp({ page, testId: ADD_TISSUE_ID });
     await selectFirstOption(page);
 
-    await clickUntilOptionsShowUp(getGeneSelectorButton, page);
+    await clickUntilOptionsShowUp({ page, testId: ADD_GENE_ID });
     await selectFirstOption(page);
 
-    const filtersPanel = await page.$("*css=div >> text=Filters");
+    const filtersPanel = page.getByTestId("filters-panel");
```

Comment:
we still need await here I think?

Reply:
no await necessary! locators aren 't async.